Makefile

```diff
@@ -163,6 +163,12 @@ GOLANGCI_LINT_VERSION := v1.60.3
 GOLANGCI_LINT := $(LOCALBIN)/golangci-lint-$(GOLANGCI_LINT_VERSION)
 $(GOLANGCI_LINT): $(LOCALBIN)
 	$(call go-install-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/cmd/golangci-lint,$(GOLANGCI_LINT_VERSION))
+	
+# Don't get confused, there is a single linter called gci, and the mega linter we use is called golangci-lint.
```

Comment:
nit: I think the comment might be confusing now since nowhere is `gci` called a linter (and it isn't; not even its own README calls it a linter)

Reply:
still related, but I called out the relationship better.